integration-tests/test/wait.py

```diff
@@ -140,7 +141,7 @@ def is_satisfied(self) -> bool:
         return actual_blocks_count >= self.blocks_count
 
 
-def wait_on_using_wall_clock_time(predicate: PredicateProtocol, timeout: int) -> None:
+def wait_using_wall_clock_time_or_fail(predicate: PredicateProtocol, timeout: int) -> None:
```

Comment:
I suggested it the other way around.  This function should be called `wait_using_wall_clock_time` because throwing an exception is the default approach, and the other one `wait_using_wall_clock_time_or_fail` because calling `fail` isn't the default approach.